dbos/__init__.py

```diff
@@ -19,6 +20,7 @@
     "WorkflowStatus",
     "WorkflowStatusString",
     "load_config",
+    "process_config",
```

Comment:
We can discuss the relevance of exporting this. I think its generally helpful for users writing integration tests.

Reply:
Don't export it. Exporting `load_config` was already a mistake, which we'll have to pay for in this PR...